egs2/aishell/asr1/run.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+stage=1
+stop_stage=11
+ngpu=1
+nj=16
+
+train_set=train
+dev_set=dev
+eval_sets="test "
+
+asr_config=conf/train_asr_rnn.yaml
+decode_config=conf/decode_asr_rnn.yaml
+
+lm_config=conf/lm.yaml
+use_lm=true
+use_wordlm=false
+
+# speed perturbation related
+speed_perturb=true  # train_set will be "${train_set}_sp" if speed_perturb = True
+perturb_factors="0.9 1.0 1.1"
+
```

Comment:
The original format of AISHELL is wav but the default is flac, so would you specify ``--audio_type wav`` to avoid converting of wave files?

Reply:
OK, I will fix this.